consensus/src/network.rs

```diff
@@ -25,7 +25,7 @@ impl<H: Hasher, D: Data, S: Signature, MS: PartialMultisignature> NetworkDataInn
 /// NetworkData is the opaque format for all data that a committee member needs to send to other nodes.
 #[derive(Clone, Debug)]
 pub struct NetworkData<H: Hasher, D: Data, S: Signature, MS: PartialMultisignature>(
-    pub(crate) NetworkDataInner<H, D, S, MS>,
+    pub NetworkDataInner<H, D, S, MS>,
```

Comment:
Welp, that is an actual problem. We _really_ don't want this to be public, or even its very type. Not sure what should be done about this, but it might mean that removing the `(crate)`s is too much work for this PR.